sdk/core/Azure.Core/src/Pipeline/RetryPolicy.cs

```diff
@@ -21,7 +21,7 @@ public class RetryPolicy : HttpPipelinePolicy
         /// <summary>
         /// Gets the delay to use for computing the interval between retry attempts.
         /// </summary>
-        protected Delay Delay { get; }
+        private readonly Delay _delay;
```

Comment:
Is the reason we make this no longer protected because the only place subclasses would have accessed it was in the GetNextDelay method?

Reply:
Yep, it's no longer useful for implementers as they cannot influence the delay that is returned besides the delay instance they pass in.

Reply:
Cool - that's makes sense! + a nice additional simplification of the public API surface :)